sql/003_create_parents.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE `parents` (
+  `idparents` int NOT NULL,
+  `telephone` varchar(45) NOT NULL,
+  `email` varchar(45) NOT NULL,
+  `parent_name` varchar(45) NOT NULL,
+  `parents_child` varchar(45) NOT NULL,
```

Comment:
How do we find out which parent is related to which volunteer? Why don't we replace parents_child with a foreign key that points to `volunteers` table?